--- route53.rb	2013-11-18 08:21:04.015019761 -0500
+++ route53.rb	2013-11-18 08:21:04.015019761 -0500
@@ -345,6 +345,10 @@
             }
           else
             record.ResourceRecords { |resources|
+            ### DPH - Accomodate Ruby-2+Builder-3.2.2 Returning a String instead of Array
+            ###       Ruby-1.8/1.9+Builder-x Returned an Array, so probably need this hack
+            ###       for backward compatibiltiy?
+            puts "\nDEBUG: @values ["+@values.class.to_s+"]\n"
+            if (@values.class.to_s == "String") then @values = [@values] end
+            ### / DPH
               @values.each { |val|
                 resources.ResourceRecord { |record|
                   record.Value(val)
